src/pages/gateway/migration.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Migration
+description: This page describes breaking changes and how to migrate from one version of API Mesh for Adobe Developer App Builder to another.
+---
+
+# Migration
+
+This page lists breaking changes between versions of API Mesh for Adobe Developer App Builder. For a list of new features and resolved issues, refer to the [release notes](release-notes.md).
+
+To update to the latest version, run the following command:
+
+```bash
+npm install -g @adobe/aio-cli-plugin-api-mesh
+```
+
+See our [NPM page](https://www.npmjs.com/package/@adobe/aio-cli-plugin-api-mesh) or [Getting Started](getting-started.md#configure-your-environment) for more information.
+
+## Migrating from Beta to GA
+
+If you participated in the beta for API Mesh, you may need to account for the following breaking changes when updating to the GA release.
+
+### Resolver Composition transform deprecated
+
+The Resolver Composition transform (`composeResolvers`) is no longer supported.
+
+### Hooks transform temporarily disabled
+
+The new [Hooks transform](hooks.md) will be temporarily disabled as part of this release. We will enable this transform in a future release.
+
+### Mesh create and update time
+
+While most [create](command-reference.md#aio-api-meshcreate) and [update](command-reference.md#aio-api-meshupdate) commands will happen very quickly, it can take up to five minutes to fully provision and propagate your mesh.
```

Comment:
happen very quickly - time to execute will be instantaneous 
- bare in mind that this is an async process which for sources that have big data sets can take up to minutes (5 minutes is very arbitrary - unless we want to say that''s average)
- since the big change is async - is important to say that this command won't error out anymore - you have to use the meshstatus for that - which you did add that info at the end of the next paragraph - but you can specify here that erroring out on this meshupdate changed